hera_mc/mc_session.py

```diff
@@ -1040,6 +1047,151 @@ def get_roach_temperature(self, starttime, stoptime=None, roach=None):
                                  stoptime=stoptime, filter_column='roach',
                                  filter_value=roach)
 
+    def add_node_sensor(self, time, node, top_sensor_temp, middle_sensor_temp,
```

Comment:
I think the name of this table could get a bit confusing. A function named `add_node_sensor` sounds to me like we have a database that maintains a list of sensors that have been installed on nodes, and this function would be called when we've installed a new sensor.  `add_node_sensor_readings`? `add_node_telemetry`?

Reply:
yep, I'll change the function name to be clearer.